.github/workflows/tag.yml

```diff
@@ -3,7 +3,7 @@ name: Release CD
 on:
   push:
     tags:
-      - "v*" # push to version tags trigger the build
+      - 'v*' # push to version tags trigger the build
```

Comment:
Can you revert these changes here? Replace single quote with double quote? Just to be safe. I'm talking about this file not the py files